adapters/Unity/UnityAdapter/GADMAdapterUnityBannerAd.m

```diff
@@ -57,7 +57,7 @@ - (void)loadBannerWithSize:(GADAdSize)adSize {
 
   if (![UnityAds isInitialized]) {
     NSString *gameID = [strongConnector.credentials[kGADMAdapterUnityGameID] copy];
-    [GADMAdapterUnitySingleton.sharedInstance initializeWithGameID:gameID];
+      [[GADMAdapterUnitySingleton sharedInstance] initializeWithGameID:gameID];
```

Comment:
Any reason to not use the dot notation? We typically prefer it as it's less brackets.